test/dynamo/test_utils.py

```diff
@@ -230,6 +230,7 @@ def test_dynamo_timed(self, mock_time, mock_time_ns):
  'fail_type': None,
  'fail_user_frame_filename': None,
  'fail_user_frame_lineno': None,
+ 'feature_usage': {'pytorch/remote_cache:bundle_triton_into_fx_graph_cache_v2': False},
```

Comment:
Is this gonna be different in OSS vs. fbcode? Anyway, it could be a pain if tests start failing because of JK changes. We probably want to mock or just zero out this field in the test before checking?

Reply:
This is the same in OSS vs fbcode (although I expect in OSS most people aren't consuming these metrics, so it's closer to a noop).

Reply:
But filtering this column for jk changes so that as people add it, they won't have to update this test.